tests/failing.llvm

```diff
@@ -1,11 +1,3 @@
-tests/ethereum-tests/BlockchainTests/GeneralStateTests/stCreate2/Create2Recursive.json
-tests/ethereum-tests/BlockchainTests/GeneralStateTests/stQuadraticComplexityTest/Create1000Byzantium.json
-tests/ethereum-tests/BlockchainTests/GeneralStateTests/stQuadraticComplexityTest/Create1000.json
-tests/ethereum-tests/BlockchainTests/GeneralStateTests/stRecursiveCreate/recursiveCreateReturnValue.json
-tests/ethereum-tests/LegacyTests/Constantinople/BlockchainTests/GeneralStateTests/stAttackTest/ContractCreationSpam_d0g0v0.json
-tests/ethereum-tests/LegacyTests/Constantinople/BlockchainTests/GeneralStateTests/stCallCreateCallCodeTest/Call1024PreCalls_d0g0v0.json
-tests/ethereum-tests/LegacyTests/Constantinople/BlockchainTests/GeneralStateTests/stCreate2/Create2Recursive_d0g0v0.json
-tests/ethereum-tests/LegacyTests/Constantinople/BlockchainTests/GeneralStateTests/stCreate2/Create2Recursive_d0g1v0.json
-tests/ethereum-tests/LegacyTests/Constantinople/BlockchainTests/GeneralStateTests/stQuadraticComplexityTest/Create1000Byzantium_d0g1v0.json
-tests/ethereum-tests/LegacyTests/Constantinople/BlockchainTests/GeneralStateTests/stQuadraticComplexityTest/Create1000_d0g1v0.json
-tests/ethereum-tests/LegacyTests/Constantinople/BlockchainTests/GeneralStateTests/stRecursiveCreate/recursiveCreateReturnValue_d0g0v0.json
+tests/ethereum-tests/BlockchainTests/GeneralStateTests/stBadOpcode/invalidAddr.json
+tests/ethereum-tests/BlockchainTests/GeneralStateTests/stBugs/extcodehashEmpty.json
+tests/ethereum-tests/BlockchainTests/GeneralStateTests/stReturnDataTest/modexp_modsize0_returndatasize.json
```

Comment:
I believe the red tests here were disabled because they were too slow on CI (not because they were failing), so you may need to re-disable them (if those tests haven't disappeared).

What's up with the three added green tests? Are they failing now? I guess they shouldn't be.